modal/object.py

```diff
@@ -112,19 +112,12 @@ def _new_hydrated(cls: Type[H], object_id: str, client: _Client, handle_metadata
         return obj
 
     @classmethod
-    async def from_id(cls: Type[H], object_id: str, client: Optional[_Client] = None) -> H:
-        """Get an object of this type from a unique object id (retrieved from `obj.object_id`)"""
-        # This is used in a few examples to construct FunctionCall objects
-        # TODO(erikbern): this should probably be on the provider?
-        if client is None:
-            client = await _Client.from_env()
-        app_lookup_object_response: api_pb2.AppLookupObjectResponse = await retry_transient_errors(
-            client.stub.AppLookupObject, api_pb2.AppLookupObjectRequest(object_id=object_id)
+    async def from_id(cls, object_id: str, client: Optional[_Client] = None):
```

Comment:
afaict this method isn't documented for any other usage than `FunctionCall`, so it should be fine to go straight to an exception